sources/packages/web/src/components/students/applicationTracker/EnrolmentBanner.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <application-status-tracker-banner
+    :label="enrolmentBannerDetails.bannerLabel"
+    :icon="enrolmentBannerDetails.bannerIcon"
+    :icon-color="enrolmentBannerDetails.iconColor"
+    :background-color="enrolmentBannerDetails.backgroundColor"
+  >
+    <template #content
+      ><span class="font-bold" v-if="enrolmentBannerDetails.contentHeader">{{
+        enrolmentBannerDetails.contentHeader
+      }}</span>
+      {{ enrolmentBannerDetails.content }}</template
+    >
+  </application-status-tracker-banner>
+</template>
+<script lang="ts">
+import ApplicationStatusTrackerBanner from "@/components/students/applicationTracker/generic/ApplicationStatusTrackerBanner.vue";
+import { COEStatus } from "@/types";
+import { EnrolmentBannerModel } from "@/components/students/applicationTracker/EnrolmentBanner.models";
+import { defineComponent, PropType, computed } from "vue";
+
+export default defineComponent({
+  components: {
+    ApplicationStatusTrackerBanner,
+  },
+  props: {
+    coeStatus: {
+      type: String as PropType<COEStatus>,
+      required: true,
+    },
+    coeDenialReason: {
+      type: String,
+      required: true,
+    },
+  },
+  setup(props) {
+    const enrolmentBannerDetails = computed<EnrolmentBannerModel>(() => {
+      switch (props.coeStatus) {
+        case COEStatus.completed:
+          return {
+            bannerLabel:
+              "Your enrolment has been confirmed by your institution",
```

Comment:
I can see that other statuses with the "heavy text" defined part at the template level. Is this one following a different approach or am I missing something?

Reply:
Are you talking about label? I see the label defined as string argument in all the other status as well.

![image](https://user-images.githubusercontent.com/54600590/215592010-eb1b9f05-9e56-4822-a15e-0b814a7a429b.png)

Reply:
> I can see that other statuses with the "heavy text" defined part at the template level. Is this one following a different approach or am I missing something? For instance: ![image](https://user-images.githubusercontent.com/61259237/215590589-a47d6f84-85b6-4c57-a38f-5a7d9f9cdc03.png)

About content?

Reply:
I was talking about all of them
![image](https://user-images.githubusercontent.com/61259237/215600319-1b0b1a48-178d-4a67-b28e-4dd09813adf1.png)